operations/mimir/read-write-deployment/autoscaling.libsonnet

```diff
@@ -1 +1,61 @@
-{}
+{
+  _config+:: {
+    autoscaling_mimir_read_enabled: false,
+    autoscaling_mimir_read_min_replicas: error 'you must set autoscaling_mimir_read_min_replicas in the _config',
+    autoscaling_mimir_read_max_replicas: error 'you must set autoscaling_mimir_read_max_replicas in the _config',
+  },
+
+  newMimirReadScaledObject(name, query_scheduler_container, querier_max_concurrent, min_replicas, max_replicas):: self.newScaledObject(name, $._config.namespace, {
```

Comment:
Can we just call `newQuerierScaledObject()` from this function? I would like mimir read autoscaling to mimick the querier autoscaling for now, and guarantee it. If at some point they will diverge, then we will just change the logic in `newMimirReadScaledObject()`.

Reply:
Fair enough. I did consider that but I thought we might want to tune the total read path differently. Makes sense not to duplicate until we need to though.